meta/documents/user_guide/en/initial_setup.md

```diff
@@ -82,36 +82,38 @@ This procedure is required only if [the `Send all inventory items to Wayfair` se
 1. From the main Plentymarkets page, go to `Item` >> `Edit item`
 
 2. Search for item(s) and open them
+    **Note:** Verify item variation  says **`main variation`** see image below
 
-3. **For each item**, click `Variations` and open them
+    ![item main variation](../../../images/en/item_main_variation.png)
 
-4. **For each Variation**:
 
-    1. On the `Settings` tab, make sure that the `Active` checkbox in the `Availability` Section is checked:
+3. On the `Settings` tab, make sure that the `Active` checkbox in the `Availability` Section is checked:

```

Comment:
Hm, are we missing a "For each Item" somewhere in here?

Reply:
I don't think the `for each item` is necessary